pkgs/development/libraries/inchi/default.nix

```diff
@@ -0,0 +1,48 @@
+{ pkgs, fetchurl, stdenv, unzip }:
+stdenv.mkDerivation {
+  pname = "inchi";
+  version = "1.05";
+  src = fetchurl {
+    url = "http://www.inchi-trust.org/download/105/INCHI-1-SRC.zip";
+    sha1 = "e3872a46d58cb321a98f4fd4b93a989fb6920b9c";
+  };
+
+  nativeBuildInputs = [ pkgs.unzip ];
+  outputs = [ "out" "doc" ];
+
+  enableParallelBuilding = true;
+
+  buildPhase = ''
+    (cd ./INCHI_API/libinchi/gcc && make -j$NIX_BUILD_CORES -l$NIX_BUILD_CORES)
+  '';
+  installPhase = ''
+    mkdir -p $out/lib
+    mkdir -p $out/include/inchi
+    mkdir -p $doc/share/
+
+    install -m 755 INCHI_API/bin/Linux/libinchi.so.1.05.00 $out/lib
+    ln -s $out/lib/libinchi.so.1.05.00 $out/lib/libinchi.so.1
+    ln -s $out/lib/libinchi.so.1.05.00 $out/lib/libinchi.so
+    install -m 644 INCHI_BASE/src/*.h $out/include/inchi
+
+    runHook postInstall
+  '';
+
+  postInstall = let
+    src-doc = fetchurl {
+      url = "http://www.inchi-trust.org/download/105/INCHI-1-DOC.zip";
+      sha1 = "2f54y0san34v01c215kk0cigzsn76js5";
+    };
+  in ''
+    unzip '${src-doc}'
+    install -m 644 INCHI-1-DOC/*.pdf $doc/share
+  '';
+
+  meta = with stdenv.lib; {
+    inherit version;
+    homepage = "https://www.inchi-trust.org/";
+    description = "The IUPAC International Chemical Identifier library";
+    license = licenses.lgpl2Plus;
+    platforms = platforms.all;
```

Comment:
There should also be a maintainer field.

Reply:
The other PR (nice-dcv-client #103939) that you just reviewed includes adding me to the maintainers list, so I'll fix this one up according to your suggestions (and add myself as maintainer) after we finish & merge that one.